netci.groovy

```diff
@@ -209,6 +216,13 @@ def static addTriggers(def job, def isPR, def architecture, def os, def configur
             case 'forcerelocs':
             case 'jitstress1':
             case 'jitstress2':   
+            case 'js2_jsr1':
```

Comment:
I just wanted to make sure there is no path length problem and check functionality before using non-abbreviated names. If you think that using long ones in this checkin is better, I will update them.